README.md

```diff
@@ -21,23 +21,30 @@ All supported workflows are available in the `workflows` directory in the projec
 Prerequisites and installation
 ==============================
 
-Piper has been tested on the Java(TM) SE Runtime Environment (build 1.7.0_25) on the [UPPMAX](http://www.uppmax.uu.se) cluster Milou. It might run in other environments, but this is untested. Besides the JVM, Piper depends on [Maven (version 3+)](http://maven.apache.org/) for building (the GATK), [Make](http://www.gnu.org/software/make/) to install, and [git](http://git-scm.com/) to checkout the source. To install piper, make sure that these programs are in you path and then clone this repository and run the setup script:
+Piper has been tested on the Java(TM) SE Runtime Environment (build 1.7.0_25) on the [UPPMAX](http://www.uppmax.uu.se) cluster Milou. It might run in other environments, but this is untested. Besides the JVM, Piper depends on [Maven (version 3+)](http://maven.apache.org/) for building (the GATK), [SBT](http://www.scala-sbt.org/) and [git](http://git-scm.com/) to checkout the source. To install piper, make sure that these programs are in you path and then clone this repository and run the setup script:
 
-    git clone https://github.com/Molmed/piper.git
+    git clone https://github.com/NationalGenomicsInfrastructure/piper.git
     cd piper
-    ./setup.sh <path to install Piper to. Default is: $HOME/Bin/Piper>
-    # Follow the instructions printed by the setup script.
+    sbt universal:packageBin
+    # This will create two zip files.
+    # Unzip these where you wish to install piper
+    ./piper/target/universal/piper-v[version].zip
+    ./setup-file-creator/target/universal/setupfilecreator-v[version].zip
```

Comment:
Are the zip-file extraction interactive or is it necessary to specify a location on the command line?